cPP_DSC.adoc

```diff
@@ -1975,9 +1975,9 @@ _If both [.underline]#Root of Trust for Measurement# and [.underline]#Root of Tr
 
 FPT_ROT_EXT.2 Root of Trust for Storage
 
-FPT_ROT_EXT.2.1:: The TSF shall prevent unauthorized access to SDOs associated with the Root of Trust for Storage.
+FPT_ROT_EXT.2.1:: The TSF shall prevent unauthorized access to SDOs stored in the Root of Trust for Storage.
```

Comment:
My understanding of this SFR is that it is about the SDOs that are used to generate what we call the RoT for Storage, and not those SDOs that are protected by it. This modification makes the SFR about the SDOs that are being protected by the RoT, as opposed to the RoT itself.

I don't this this is needed

Reply:
Fair enough, but then we should also change it in the Application Note, no?

Reply:
Yeah, that would make sense to change it from stored in to associated with in the app note as well. I think that most of the changes you had there could be dropped in favor of that particular change.

Reply:
@woodbe it is changed in the app note